cookbook/profiler/data_collector.rst

```diff
@@ -46,106 +46,151 @@ access to in local properties.
-following example can help you get started:
+The information collected by your data collector can be displayed both in the
+web debug toolbar and in the web profiler. To do so, you need to create a Twig
+template that includes some specific blocks.
+
+In the simplest case, you just want to display the information in the toolbar
+without providing a profiler panel:
 
 .. code-block:: jinja
 
-    {% extends 'WebProfilerBundle:Profiler:layout.html.twig' %}
+    {% extends '@WebProfiler/Profiler/layout.html.twig' %}
 
     {% block toolbar %}
-        {# This toolbar item may appear along the top or bottom of the screen.#}
         {% set icon %}
-        <span class="icon"><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABoAAAAcCAQAAADVGmdYAAAAAmJLR0QA/4ePzL8AAAAJcEhZcwAACxMAAAsTAQCanBgAAAAHdElNRQffAxkBCDStonIVAAAAGXRFWHRDb21tZW50AENyZWF0ZWQgd2l0aCBHSU1QV4EOFwAAAHpJREFUOMtj3PWfgXRAuqZd/5nIsIdhVBPFmgqIjCuYOrJsYtz1fxuUOYER2TQID8afwIiQ8YIkI4TzCv5D2AgaWSuExJKMIDbA7EEVhQEWXJ6FKUY4D48m7HYU/EcWZ8JlE6qfMELPDcUJuEMPxvYazYTDWRMjOcUyAEswO+VjeQQaAAAAAElFTkSuQmCC" alt=""/></span>
-        <span class="sf-toolbar-status">Example</span>
+            <span class="icon"><img src="..." alt=""/></span>
+            <span class="sf-toolbar-status">Information</span>
         {% endset %}
 
         {% set text %}
-        <div class="sf-toolbar-info-piece">
-            <b>Quick piece of data</b>
-            <span>100 units</span>
-        </div>
-        <div class="sf-toolbar-info-piece">
-            <b>Another quick thing</b>
-            <span>300 units</span>
-        </div>
+            <div class="sf-toolbar-info-piece">
+                <b>Quick piece of data</b>
+                <span>100 units</span>
+            </div>
+            <div class="sf-toolbar-info-piece">
+                <b>Another piece of data</b>
+                <span>300 units</span>
+            </div>
         {% endset %}
 
-        {# Set the "link" value to false if you do not have a big "panel"
-           section that you want to direct the user to. #}
-        {{ include('@WebProfiler/Profiler/toolbar_item.html.twig', { 'link': true }) }}
+        {# the 'link' value set to 'false' means that this panel doesn't show
+           a section in the web profiler. #}
+        {{ include('@WebProfiler/Profiler/toolbar_item.html.twig', { link: false }) }}
+    {% endblock %}
+
+The ``toolbar`` block is used to create a panel in the web debug toolbar. The
+inner ``icon`` block defines the content to display right inside the toolbar
+and the ``text`` block defines the content displayed when hovering the mouse
+over that same panel.
+
+.. tip::
+
+    Built-in collector templates define all their images as embedded base64-encoded
+    images. This makes them work everywhere without having to mess with web assets
+    links:
+
+    .. code-block:: html
+
+        <img src="data:image/png;base64,..." />
+
+    You are encouraged to use this same technique for your own toolbar panels.
```

Comment:
In symfony 2.7, we decided that using SVG icons was better than PNG ones. This was not backported to older versions because it was not considered as a bugfix, but I would encourage using SVG in the doc